eng/pipelines/runtime-cet.yml

```diff
@@ -0,0 +1,131 @@
+
+trigger:
+  batch: true
+  branches:
+    include:
+    - release/*.*
+  paths:
+    include:
+    - '*'
+    - docs/manpages/*
+    exclude:
+    - /**/*.md
+    - eng/Version.Details.xml
+    - .github/*
+    - docs/*
+    - LICENSE.TXT
+    - PATENTS.TXT
+    - THIRD-PARTY-NOTICES.TXT
+
+
+variables:
+  - template: /eng/pipelines/common/variables.yml
+
+jobs:
+
+#
+# Evaluate paths
+#
+# - ${{ if eq(variables.dependOnEvaluatePaths, true) }}:
+#   - template: /eng/pipelines/common/evaluate-default-paths.yml
+
+#
+# Build CoreCLR checked
+# Only when CoreCLR is changed
+#
+- template: /eng/pipelines/common/platform-matrix.yml
+  parameters:
+    jobTemplate: /eng/pipelines/coreclr/templates/build-job.yml
+    buildConfig: checked
+    platforms:
+    - windows_x64
+    jobParameters:
+      testGroup: innerloop
+      condition: >-
+        true
+#         or(
+#           eq(dependencies.evaluate_paths.outputs['SetPathVars_coreclr.containsChange'], true),
+#           eq(dependencies.evaluate_paths.outputs['SetPathVars_runtimetests.containsChange'], true),
+#           eq(variables['isRollingBuild'], true))
+          
+#
+# Build CoreCLR release
+# Always as they are needed by Installer and we always build and test the Installer.
+#
+- template: /eng/pipelines/common/platform-matrix.yml
+  parameters:
+    jobTemplate: /eng/pipelines/coreclr/templates/build-job.yml
+    buildConfig: release
+    platforms:
+    - windows_x64
+    jobParameters:
+      testGroup: innerloop
+
+
+- template: /eng/pipelines/common/platform-matrix.yml
+  parameters:
+    jobTemplate: /eng/pipelines/libraries/build-job.yml
+    buildConfig: release
+    platforms:
+    - windows_x64
+    jobParameters:
+      testScope: innerloop
+
+#
+# Libraries release build that only runs when coreclr is changed in PRs
```

Comment:
Apologies for the confusion, the net effect of the changes I suggested and you made is that this second build of libraries is no longer needed and is in fact confusing the infra because now it thinks there are two identical library builds. I would suggest removing lines 75 to 88.